src/calendar/index.njk

```diff
@@ -2,44 +2,121 @@
 title: Calendar
 layout: layouts/base.njk
 ---
-<div class="grid gap-4">
-  <h2>Events this week</h2>
 
+<div class="grid grid-flow-row gap-3">
+  <h2 class="text-xl mt-5 mx-4">EVENTS THIS WEEK</h2>
```

Comment:
Can you use the CSS text-transform here rather than 'hardcoding' the capitals here please? It's better for accessibility

Ref: https://tailwindcss.com/docs/text-transform#transforming-text